From: Antonio Rojas <nqn1976@gmail.com>
Date: Sat, 19 Sep 2015 22:14:14 +0000
Subject: Allow overriding Qt designer plugins install dir
X-Git-Tag: v15.11.80
X-Git-Url: http://quickgit.kde.org/?p=marble.git&a=commitdiff&h=f59d559347fea1e2ada00e111f1faaaa74151614
---
Allow overriding Qt designer plugins install dir

Allow specifying a custom QT_PLUGINS_DIR in the cmake command and don't
overwrite it.

Testing Done:
Builds and installs the plugins in the directory specified in the
QT_PLUGINS_DIR cmake variable.

REVIEW: 125299
CCBUG: 348554
---


--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -177,7 +177,7 @@
 add_feature_info("Qt Designer plugins" WITH_DESIGNER_PLUGIN "Marble widget support in Qt Designer. Toggle with WITH_DESIGNER_PLUGIN=YES/NO")
 
 set(EXEC_INSTALL_PREFIX  ${CMAKE_INSTALL_PREFIX} CACHE PATH  "Base directory for executables and libraries" FORCE)
-if (QT5BUILD)
+if (QT5BUILD AND NOT QT_PLUGINS_DIR)
    set(QT_PLUGINS_DIR ${CMAKE_INSTALL_LIBDIR}/plugins)
 endif()
 

